rasa/utils/tensorflow/model_data.py

```diff
@@ -398,12 +398,13 @@ def number_of_examples(self, data: Optional[Data] = None) -> int:
         if not example_lengths:
             return 0
 
+        # TODO: See, why this needs to be commented
```

Comment:
@dakshvar22 seems this needs to be commented as the default actions have an `action_name` such as `action_listen` while the e2e bot utterances use `action_text`. If you have 1000 actions, you end up with 11 or so default actions and 989 e2e actions. And their ids get sorted into different buckets through the masks. That's why then lengths don't work out.